tests/test_exchanges_json.py

```diff
@@ -2,18 +2,17 @@
 
 from electricitymap.contrib.config import EXCHANGES_CONFIG, ZONES_CONFIG
 
-ZONE_KEYS = ZONES_CONFIG.keys()
-
 
 class ExchangeJsonTestcase(unittest.TestCase):
     def test_all_zones_in_zones_json(self):
+        zone_keys = ZONES_CONFIG.keys()
```

Comment:
I don't think this should be moved since it's a constant?

Reply:
Are you concerned about moving ZONE_KEYS, or using ZONES_CONFIG?

If the first: and nothing imports (and nothing _should_ import!) the test to use a global in a test module. If it's a constant used throughout one test case, it could be a class-level attribute in the test case class. It was only used in one test method, which is why I moved it here.

If the second: If we're concerned about tests (or other code elsewhere) potentially mutating ZONES_CONFIG, we should use `frozendict` when creating the config dicts.

Reply:
Sounds good!

If you wish you can open a "RFC" issue about using frozendict for the config dicts. I don't think there should be an issue with that but best to discuss it first.

Reply:
Thanks -- and indeed I think using frozendict could make sense here but it would be best to discuss first. Is a RFC issue just an issue (created with https://github.com/electricitymaps/electricitymaps-contrib/issues/new) with "RFC: ..." in the title?